--- origsrc/unac-1.8.0.orig/Makefile.am	2004-10-16 10:26:11.000000000 -0500
+++ src/unac-1.8.0.orig/Makefile.am	2007-12-04 13:10:01.325498100 -0600
@@ -44,7 +44,9 @@
 	unac.h
 
 unac.c: $(UNICODEDATA) builder.in
-	$(PERL) builder -source -database=$(srcdir)/$(UNICODEDATA)
+	srcdir=$(srcdir) $(PERL) builder -source -database=$(srcdir)/$(UNICODEDATA)
+
+CLEANFILES = unac.c unac.h
 
 #
 # Tests
--- origsrc/unac-1.8.0.orig/builder.in	2004-10-16 10:26:11.000000000 -0500
+++ src/unac-1.8.0.orig/builder.in	2007-12-04 13:02:53.770926700 -0600
@@ -260,9 +260,10 @@
 #
 sub source {
     my($decomposition, $name, $verbose) = @_;
+    my($srcdir) = $ENV{'srcdir'} || '.';
 
-    my($csource) = slurp("unac.c");
-    my($hsource) = slurp("unac.h");
+    my($csource) = slurp("$srcdir/unac.c");
+    my($hsource) = slurp("$srcdir/unac.h");
 
     #
     # First step is to create the content that will be included
