diff -urN -x configure -x Makefile.in -x autom4te.cache -x ltmain.sh -x depcomp -x install-sh -x 'config.h.in*' -x 'config.*' -x aclocal.m4 -x INSTALL -x missing origsrc/unac-1.8.0.orig/Makefile.am src/unac-1.8.0.orig/Makefile.am
--- origsrc/unac-1.8.0.orig/Makefile.am	2004-10-16 10:26:11.000000000 -0500
+++ src/unac-1.8.0.orig/Makefile.am	2007-12-04 13:10:01.325498100 -0600
@@ -37,14 +37,17 @@
 # Library and headers
 #
 lib_LTLIBRARIES = libunac.la
-libunac_la_LDFLAGS = -version-info $(MAJOR_VERSION):$(MINOR_VERSION):$(MICRO_VERSION)
+libunac_la_LDFLAGS = -version-info $(MAJOR_VERSION):$(MINOR_VERSION):$(MICRO_VERSION) -no-undefined
+libunac_la_LIBADD = $(LTLIBICONV)
 libunac_la_SOURCES = unac.c
 
 include_HEADERS = \
 	unac.h
 
 unac.c: $(UNICODEDATA) builder.in
-	$(PERL) builder -source -database=$(srcdir)/$(UNICODEDATA)
+	srcdir=$(srcdir) $(PERL) builder -source -database=$(srcdir)/$(UNICODEDATA)
+
+CLEANFILES = unac.c unac.h
 
 #
 # Tests
diff -urN -x configure -x Makefile.in -x autom4te.cache -x ltmain.sh -x depcomp -x install-sh -x 'config.h.in*' -x 'config.*' -x aclocal.m4 -x INSTALL -x missing origsrc/unac-1.8.0.orig/builder.in src/unac-1.8.0.orig/builder.in
--- origsrc/unac-1.8.0.orig/builder.in	2004-10-16 10:26:11.000000000 -0500
+++ src/unac-1.8.0.orig/builder.in	2007-12-04 13:02:53.770926700 -0600
@@ -260,9 +260,10 @@
 #
 sub source {
     my($decomposition, $name, $verbose) = @_;
+    my($srcdir) = $ENV{'srcdir'} || '.';
 
-    my($csource) = slurp("unac.c");
-    my($hsource) = slurp("unac.h");
+    my($csource) = slurp("$srcdir/unac.c");
+    my($hsource) = slurp("$srcdir/unac.h");
 
     #
     # First step is to create the content that will be included
diff -urN -x configure -x Makefile.in -x autom4te.cache -x ltmain.sh -x depcomp -x install-sh -x 'config.h.in*' -x 'config.*' -x aclocal.m4 -x INSTALL -x missing origsrc/unac-1.8.0.orig/configure.ac src/unac-1.8.0.orig/configure.ac
--- origsrc/unac-1.8.0.orig/configure.ac	2004-10-16 10:26:12.000000000 -0500
+++ src/unac-1.8.0.orig/configure.ac	2007-12-04 12:46:55.084303000 -0600
@@ -49,11 +49,6 @@
 
 AM_ICONV
 
-AC_CHECK_FUNCS(iconv_open,,AC_MSG_ERROR([
-iconv_open not found try to install replacement from
-http://www.gnu.org/software/libiconv/
-ftp://ftp.gnu.org/pub/gnu/libiconv/libiconv-1.9.1.tar.gz]))
-
 AC_CHECK_FUNCS(vsnprintf)
 
 AC_LIBSOURCES(getopt.c)
